CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Various fixes & improvements
+
+- Add CLI version metadata to build upload archives for debugging purposes ([#2890](https://github.com/getsentry/sentry-cli/pull/2890))
```

Comment:
Please add more details to this changelog entry, specifically so that users understand the user-facing changes. I have been making user-readable changelog entries a priority in the last few Sentry CLI releases.